db/migrate/20220411175218_create_tasks.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.citext :title, null: false, index: { unique: true }
```

Comment:
unique in scope of the challenge

Reply:
Added validation to model here - [update task validations](https://github.com/DevChallengeIT/devchallenge-platform/pull/19/commits/81e85721125592167133b5b939a6138f74cd7af0)

Is it what you meant?